### ISSUE TITLE
Tags not getting reflected in PacBot

### ISSUE THREAD

**Issue by spider089** [tmobile/pacbot#411]
# ❔ Question

I have created tags for Application and Environment for assets in aws and changed a couple of rules to only have Application and Environment as mandatory tags. But I can still see them as non-compliant and the updated rules not taking effect since 2/3 Days.

**Comment by santhoshigorle** [comment]
@spider089  did you see new tags in the inventory??
If the data is collected it should make it complaint.

**Comment by spider089** [comment]
@santhoshigorle No. I added the missing MandatoryTags (in this case, Application and Environment) in aws and ran the rule in pacbot. Ideally, the rule violation should get removed since I've remediated it, right? The violation still persists with the same details even thought he rule moved to succeeded in Batch.
